ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1093 @@
+----
+kubectl get pods
+
+NAME                                READY   STATUS    RESTARTS   AGE
+client-0                            1/1     Running   0          15h
+postgresql-7dd74c84dc-s8sh8         1/1     Running   0          15h
+server-0                            1/1     Running   0          54m
+server-1                            1/1     Running   0          54m
+wildfly-operator-5d4b7cc868-8qhjn   1/1     Running   0          15h
+
+# check the logs of the servers are clean from any errors
+kubectl logs client-0
+kubectl logs server-0
+kubectl logs server-1
+----
+
+[#_running_on_kubernetes_verify_the_running_application]
+==== Running on Kubernetes: verify the running application
+
+After you have your applications running on Kubernetes, you can check the calls to the endpoints.
+
+[source,sh,options="nowrap"]
+----
+curl -s $(minikube service client-loadbalancer --url)/remote-outbound-stateless | jq .
+curl -s $(minikube service client-loadbalancer --url)/remote-outbound-notx-stateless | jq .
+curl -s $(minikube service client-loadbalancer --url)/direct-stateless | jq .
+curl -s $(minikube service client-loadbalancer --url)/remote-outbound-notx-stateful | jq .
+
+# to check failure resolution
+curl -s $(minikube service server-loadbalancer --url)/commits
+curl -s $(minikube service client-loadbalancer --url)/remote-outbound-fail-stateless
+# waiting for recovery is kicked on and the number of commits increases
+while true; do
+  curl -s $(minikube service server-loadbalancer --url)/commits
+  echo
+  sleep 2
+done
+----
+
+=== Running on OpenShift
+
+include::../shared-doc/cd-openshift-getting-started.adoc[leveloffset=+3]
+
+Running on https://www.okd.io[OpenShift] is similar to running on Kubernetes.
+The only difference is that some processes are a bit more automated.
+
+Before starting to implement the QS on OpenShift, review <<_running_on_kubernetes, Running on Kubernetes>>.
+
+==== Running on OpenShift: Prerequisites
+
+include::../shared-doc/cd-create-project.adoc[leveloffset=+4]
+
+[#_running_on_openshift_start_postgresql_database]
+===== Running on OpenShift: Start PostgreSQL database
+
+For testing purposes you can use
+
+[source,sh,options="nowrap",subs="+quotes,attributes+"]
+----
+# change path to ${PATH_TO_QUICKSTART_DIR}
+cd ${PATH_TO_QUICKSTART_DIR}
+# deploy not-production ready PostgreSQL database to OpenShift
+oc create -f ejb-txn-remote-call/client/extensions/postgresql.deployment.yaml
+----
+
+[#_running_on_openshift_build_the_application]
+==== Running on OpenShift: Build the application
+
+OpenShift provides the s2i functionality out-of-the-box.
+The quickstart defines two OpenShift templates.
+ The `BuildConfig` template builds the application based on the images
+configured in the `ImageStream` template. The `BuildConfig` template is named `wildfly-s2i`.
+
+[source,sh,options="nowrap",subs="+quotes,attributes+"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}
+
+# Define the image streams of app server images.
```

Comment:
You should try to use the s2i supported process instead, which is already documented in the cd-* or xp-* asciidoc sections in /shared-docs. The microprofile-rest-client Quickstart also creates multiple OpenShift apps, so perhaps you can reuse the solution where specific content for your quickstart is added in the shared-docs sections, and rendered if an attribute is set. @xstefank is the author of such quickstart, and usually is happy to help :-)